src/app/utils/render.js

```diff
@@ -0,0 +1,18 @@
+const render = (parentId, ...children) => {
+  if (!children || !parentId) {
+    throw new Error(
+      'In params must have been added parentId and at least one child to render in it',
+    );
+  }
+
+  const parentElem = document.getElementById(parentId);
+  if (!parentElem) {
+    throw new Error(`Not found element with id ${parentId}`);
+  }
```

Comment:
```suggestion
const parentElem = document.getElementById(parentId);
const validated = [children, parentId, parentElem ]

if (validated.filter(el => el == false)) {
throw new Error(
      'Wrong parameter set in validated []'
    );
}
```

Reply:
It doesn't seem to be working properly

Reply:
that's right. I hoped you could suggest a quick fix 🔨 

Consider an empty array being true and unknown num of children

Reply:
Or move along if needed. That's not a big deal, it's just a suggestion